### DISCUSSION TITLE
Stop Publication Service not working (Preview)

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#965] [Q&A] [answered]
We are trying to test the stop publication service in preview and it does not work. It gives the following error:

_CloudFront attempted to establish a connection with the origin, but either the attempt failed or the origin closed the connection.
We can't connect to the server for this app or website at this time. There might be too much traffic or a configuration error. Try again later, or contact the app or website owner._

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/a7c841c3-d061-4dba-987d-03d699f89533)

Is it a temporary indence?

Thanks in advance

**Comment by YvesJo** [comment, marked as answer]
Hi,
Your request has been received by the HD who will answer it, so I'm closing this non-SDK related concern.
KR
